h/h_api/model/json_api.py

```diff
@@ -0,0 +1,34 @@
+"""Generic model objects representing objects in JSON API style."""
+
+from h.h_api.model.base import Model
+
+
+class JSONAPIError(Model):
+    """A JSON API error wrapper."""
+
+    @classmethod
+    def create(cls, error_bodies):
+        return cls({"errors": [cls.extract_raw(body) for body in error_bodies]})
```

Comment:
I'm not sure why `create()` is being used here instead of `__init__()`?

Reply:
It'll be more obvious in the next PR when more model classes get introduced, but basically the `__init__` method is used for reading from a raw data structure, and is completely consistent across all children of `Model`. 

The `create` method is always model specific, and takes named arguments to construct the object and provides the interface to a caller who (theoretically) doesn't know the data structure.